### PR TITLE
Add storage entities review

### PR DESCRIPTION
**Pull request by yolo3301**
Sending one time warning comment if entity code gets changed.

### CODE REVIEW

File: vsts/reviewStorageEntities.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	"strings"
+)
+
+type storageEntitiesReview struct {
+	diffs       *diffs
+	pullRequest *PullRequest
+}
+
+func (r *storageEntitiesReview) getBotCommentPrefix() string {
+	return "[BOT_Image]\n"
```
Comment (wenwu449) [inline vsts/reviewStorageEntities.go]: use a unique prefix

File: vsts/reviewStorageEntities.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+}
+
+func (r *storageEntitiesReview) getCommentContent(changedStorageEntityPathes []string) string {
+	return fmt.Sprintf(
+		"%s:warning:\nThe following storage entities were changed that could potentially break back compatibility: **%+v**",
```
Comment (wenwu449) [inline vsts/reviewStorageEntities.go]: make message actionable?

File: vsts/reviewStorageEntities.go [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+	}
+
+	if commentThread.Status == "" {
+		commentContent := r.getCommentContent(changedStorageEntityPathes)
+		err := createCommentThread(r.pullRequest.Resource.PullRequestID, changedStorageEntityPathes[0], 1, commentContent)
+		if err != nil {
+			return false, err
+		}
+
+		// Only return false when creating the comment for the first time.
```
Comment (wenwu449) [inline vsts/reviewStorageEntities.go]: minor: print a log for the file for easier troubleshooting

File: vsts/reviewStorageEntities.go [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+					commentThread = thread
+					break
+				}
+			}
+		}
+	}
+
+	if commentThread.Status == "" {
+		commentContent := r.getCommentContent(changedStorageEntityPathes)
+		err := createCommentThread(r.pullRequest.Resource.PullRequestID, changedStorageEntityPathes[0], 1, commentContent)
```
Comment (wenwu449) [inline vsts/reviewStorageEntities.go]: only comment on one file?

File: vsts/reviewStorageEntities.go (hunk 3 above)
Comment (yolo3301) [reply vsts/reviewStorageEntities.go]: It's already printed out on line 47

**Pull request merged**